rts/Sim/Units/UnitTypes/Builder.cpp

```diff
@@ -515,12 +515,11 @@ bool CBuilder::UpdateCapture(const Command& fCommand)
 	if (curCapturee->captureProgress < 1.0f)
 		return true;
 
-	if (!curCapturee->ChangeTeam(team, CUnit::ChangeCaptured)) {
-		// capture failed
-		if (team == gu->myTeam) {
-			LOG_L(L_WARNING, "%s: Capture failed, unit type limit reached", unitDef->humanName.c_str());
-			eventHandler.LastMessagePosition(pos);
```

Comment:
Is the removal of a callin invocation intended? (the new one does not contain it)

Reply:
Removing `LastMessagePosition` is probably ok, adding `UnitTaken` isn't. Generally leave the UnitBla callins as they were.

Reply:
The goal was to replace `UnitTaken` with the `reason`, since they are redundant.